src/internal_events/aws_sqs.rs

```diff
@@ -1,12 +1,12 @@
 use metrics::counter;
-#[cfg(feature = "sources-aws_s3")]
+#[cfg(any(feature = "sources-aws_s3", feature = "sources-aws_sqs"))]
 pub use s3::*;
 use vector_core::internal_event::InternalEvent;
 
 #[cfg(any(feature = "sources-aws_s3", feature = "sources-aws_sqs"))]
 use crate::internal_events::prelude::{error_stage, error_type};
 
-#[cfg(feature = "sources-aws_s3")]
```

Comment:
It seems like `SqsMessageReceiveError` is the only new one being used by the SQS source. Maybe it could just be moved out of `mod s3` since it is used by both now?